libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/MapboxTripStarterExtra.kt

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.navigation.core.trip
+
+import androidx.annotation.StringDef
+import com.mapbox.navigation.core.directions.session.RoutesObserver
+
+object MapboxTripStarterExtra {
+
+    /**
+     * The [MapboxTripStarter] will use the best device location for a trip session.
+     */
+    const val MAPBOX_TRIP_STARTER_FOLLOW_DEVICE = "MAPBOX_TRIP_STARTER_FOLLOW_DEVICE"
+
+    /**
+     * The [MapboxTripStarter] will enable replay for the navigation routes.
+     */
+    const val MAPBOX_TRIP_STARTER_REPLAY_ROUTE = "MAPBOX_TRIP_STARTER_REPLAY_ROUTE"
```

Comment:
I'm making this scalable because I'd like to plan for adding `MAPBOX_TRIP_STARTER_REPLAY_HISTORY`

We should be able to add a path to a history file and then replay it. I'm also, considering adding it right now because it is probably simple